.github/workflows/benchmarks.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           make clean || true
           pip uninstall pennylane-lightning -y
           cd ../pr
-          pip install -e .
+          CXX=g++-10 pip install -e .
```

Comment:
Do we need to explicitly list g++-10 here explicitly? Would it be safer to specific this as a minimum supported version in the CMake, and assume that the system has >=10 available?

Since Ubuntu 22.04 defaults to GCC 11 and manylinux2014 defaults to GCC 10, I think that would likely be a better option.

Reply:
I set `cmake` to raise an error if the GCC version is less than 9 (i.e. >= 10). Still, as it is not straightforward to use the latest installed version in a workflow (and I think it is safer to fix the GCC version for both tests and build wheels), I just set